packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -183,12 +185,14 @@ const DialpadContainer = (props: {
   onDisplayDialpadInput?: (input: string) => string;
   /**  on change function for text field */
   onChange?: (input: string) => void;
+  /**  boolean input to determine show/hide delete button, default true */
+  showDeleteButton?: boolean;
   styles?: DialpadStyles;
 }): JSX.Element => {
   const theme = useTheme();
   const [textValue, setTextValue] = useState('');
 
-  const { onSendDtmfTone, onClickDialpadButton, onDisplayDialpadInput, onChange } = props;
+  const { onSendDtmfTone, onClickDialpadButton, onDisplayDialpadInput, onChange, showDeleteButton } = props;
```

Comment:
Did you forget to assign a default value of `true` to `showDeleteButton` here?

Reply:
Added default value down below in export const Dialpad = (props: DialpadProps): JSX.Element => {